### ISSUE TITLE
Hot Module Reloading not working

### ISSUE THREAD

**Issue by chrischen** [TrueCar/gluestick#1182]
I updated Gluestick and now HMR does not seem to be working. The client is connecting to the HMR server and is pulling the updates (syntax errors will cause an update on the page still).

```
gluestick-cli: 2.1.0 (global binary)
gluestick: 2.1.0 (local dependency in current directory)
```

Accessing via
http://localhost:8888

In the browser:
```
[HMR] bundle rebuilding
client.js:237 [HMR] bundle rebuilt in 608ms
```

However no re-render is triggered. Refreshing the page will load new page.

I tested this on a fresh gluestick app. OS is Mac OS High Sierra and tested on the latest versions of Chrome and Safari.

**Comment by saltycrane** [comment]
When I upgraded `react-hot-loader` to a newer version in my local environment, hot loading worked better. Can gluestick change the version of `react-hot-loader` from `3.0.0-beta.6` to a newer version? (I see `4.1.1` on npm currently.)

**Comment by chrischen** [comment]
I updated to 4.1.1 and hot-reloading still doesn't seem to be working. It's connecting to HMR server and reporting bundle rebuilt, but react is not re-rendering the components still.

UPDATE: Seems a fresh gluestick project works once I updated to react-hot-loader 4.1.1, but my project still doesn't work. Probably some other package needs to be updated.

UPDATE: deleted node_modules and reinstalled along with 4.1.1 of react-hot-loader and it's all working now.
